guides/v2.1/extension-dev-guide/framework/array-manager.md

```diff
@@ -0,0 +1,134 @@
+---
+group: php-developer-guide
+title: Array Manager
+subgroup: Framework
+menu_title: Array Manager
+menu_order: 1005
+---
+
+## Overview
+
+The [`Magento\Framework\Stdlib\ArrayManager`]({{ site.mage2bloburl }}/{{ page.guide_version }}/lib/internal/Magento/Framework/Stdlib/ArrayManager.php){:target="_blank"} class provides a better way of managing the nested arrays. 
+Being more specifically, heavily nested associative arrays. It is used primarily when handling data from UiComponents within [DataProviders]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_data_source.html) and [Modifiers]({{ page.baseurl }}/ui_comp_guide/concepts/ui_comp_modifier_concept.html), which, are actually part of a complicated process of parsing XML files as associative arrays.
```

Comment:
I suggest combining this info with the previous paragraph, as shown in the example I provide above.